command.go

```diff
@@ -138,6 +142,28 @@ func ExecuteHide(c Command, v *VHS) {
 	v.PauseRecording()
 }
 
+// ExecuteRequire is a CommandFunc that checks if all the binaries mentioned in the
+// Require command are present. If not, it exits with a non-zero error.
+func ExecuteRequire(c Command, v *VHS) {
+	unavailableBinaries := make([]string, 0)
+
+	requiredBinaries := strings.Split(c.Args, ",")
```

Comment:
I don't feel very good about this function as it's not inline with the actual `Require` syntax. 

Should we just move this logic to the `checkRequires` function?

Reply:
I agree, I've changed it so that all the logic is more maintainable by introducing runtime errors on the VHS struct.